WebService/v1/Controllers/DeviceModelPropertiesController.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
```

Comment:
Instead of adding a separate controller for DeviceModelsProperties, I suggest we should follow this pattern -
"/items?{action}" and create an action "getProperties" in DeviceModelsController.
@dluc , what are your thoughts on using querystring to specify this as an action for getting properties ?

Reply:
This is something we discussed and the current solution is a compromise. For consistency we decided to have a new read-only resource type called "DeviceProperty". The action pattern you mention would work too, and is similar to what the actual solution would be, i.e. getting the list of models, excluding all the fields except the properties, something like:

```
GET /v1/deviceModels?$select=Properties
```

We haven't designed this yet though, so we went with `GET /v1/deviceModelsProperties` for the time being, which is close enough.

Another approach would have been using `GET /v1/deviceModels` and having some logic in the UI to extract the list, but that would have couple the UI with the backend too much in my opinion.

Reply:
👍